app/src/main/java/edu/wisc/ece/pinpoint/data/OrderedHashSet.java

```diff
@@ -18,7 +18,7 @@ public OrderedHashSet() {
     }
 
     /**
-     * Add the specified element at the end of the list if it doesn't already exist in O(1).
+     * Add the specified element at the start of the list if it doesn't already exist in O(1).
```

Comment:
doesn't `add` for `ArrayList`'s append to the end of a list?

additionally, I think the `remove` javadoc within `OrderedHashSet` is incorrect, `remove` is an `O(n)` operation for Lists.

Reply:
I reversed the indexed access in the `get()` method, so adds to the end of the ArrayList are effectively at index 0 as exposed by the `get()` method.

You are right about the `remove()` method though so I fixed the javadoc.